src/hammer-vlsi/constraints_test.py

```diff
@@ -101,6 +102,92 @@ def test_optional_generated(self) -> None:
             divisor=None
         ))
 
+class BumpsTest(unittest.TestCase):
+    def test_bump_naming(self) -> None:
+        assignments = [
+            BumpAssignment(name="foo",no_connect=False,x=Decimal(1),y=Decimal(1),group=None,custom_cell=None),
```

Comment:
Minor nitpick: no need to fix this now, but at some point we may run a style formatter that will format these automatically (e.g. instead of `x=1,y=1`, `x = 1, y = 1`, etc)

Reply:
You have previously asked for no spaces around `=`, so I'm not sure what the concern is (spaces after comma?), but I agree we should eventually just have a style formatter as part of the process. Until then, I don't think it's useful to spend a lot of time here.

Reply:
Oops, my bad on the `=` thing - I think I am just getting at the space after comma.